modules/png-extraction/ImageExtractor.py

```diff
@@ -501,10 +508,7 @@ def execute(pickle_file, dicom_home, output_directory, print_images, print_only_
     for mapping in mappings:
         map_list.append(pd.read_csv(mapping,dtype='str'))
     merged_maps = pd.concat(map_list,ignore_index=True)
-    if print_only_common_headers:
-        mask_common_fields = merged_maps.isnull().mean() < 0.1
```

Comment:
Not sure how you can achieve "CommonHeadersOnly=True" by removing these 4 lines that check and verify a header appears in at least 10% of the entries.

Reply:
In line 505, there should be merged_meta instead of merge_map ,because we are dealing with header not with data mapping.

Reply:
In line 506, we need list of those headers.

Reply:
I think @ramon349 may have some more insights.